.pre-commit-config.yaml

```diff
@@ -49,10 +37,6 @@ repos:
 - repo: https://github.com/pre-commit/pygrep-hooks
   rev: v1.10.0
   hooks:
-  - id: python-check-blanket-noqa
-  - id: python-check-blanket-type-ignore
-  - id: python-no-eval
-  - id: python-use-type-annotations
```

Comment:
I take it that these are covered by ruff?

Reply:
Yep, the PGH checks. https://github.com/charliermarsh/ruff#pygrep-hooks-pgh